### ISSUE TITLE
i7-8809G was added to the supported list.

### ISSUE THREAD

**Issue by nosirrahx** [rcmaehl/WhyNotWin11#598]
https://docs.microsoft.com/en-us/windows-hardware/design/minimum/supported/windows-11-supported-intel-processors

This is one of those "technically 7th gen but labeled 8th gen" CPUs. 

Microsoft's PC Health Check now passes this CPU but WhyNotWin11 still lists it as unsupported.

**Comment by micwoj92** [comment]
That CPU is already added , use latest dev version.
https://github.com/rcmaehl/WhyNotWin11/blob/7eda51dfd73aef5672f5d812aafe5eecde12f9fe/includes/SupportedProcessorsIntel.txt#L105

**Comment by nosirrahx** [comment]
I had to delete the old \AppData\Local\WhyNotWin11\SupportedProcessorsIntel.txt for this to work and I think I know why.

At the top of the AMD file is '2.4.1.0' but at the top of the Intel file is '2.4.0.0'. I suspect that since the updated list and last list share the same version that your app did not update the Intel list when I ran the new build until I manually deleted SupportedProcessorsIntel.txt first.
